tests/test_all_fields.py

```diff
@@ -548,8 +572,22 @@ def all_fields_test(self, streams_to_test):
                                         f"AssertionError({failure_1})")
 
                                     nested_key = KNOWN_NESTED_MISSING_FIELDS.get(stream, {})
-                                    if self.find_nested_key(nested_key, expected_field_value, field):
-                                        continue
+                                    # Check whether expected_field_value is list or not.
+                                    # If expected_field_value is list then loop through each item of list
+                                    if type(expected_field_value) == list:
```

Comment:
Move this to the separate function with comments?

Reply:
Added this code to a separate function.